### PR TITLE
[13.0][MIG] web_disable_export_group: Migration to 13.0 

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa
TT26186

### CODE REVIEW

**Comment by joao-p-marques** [comment]
@chienandalu and @Tardo can you please review?

File: web_disable_export_group/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -6,3 +6,4 @@
 * `Tecnativa <https://www.tecnativa.com>`_:
 
   * David Vidal <david.vidal@tecnativa.com>
```
Comment (chienandalu) [inline web_disable_export_group/readme/CONTRIBUTORS.rst]: You can delete the email here

**Review by chienandalu — CHANGES_REQUESTED** [review]
Functional test ok

Check pre-commit errors and a minor comment:

File: web_disable_export_group/static/src/js/web_disable_export_group_tour.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    var tour = require("web_tour.tour");
+
+    var _t = core._t;
+
+    tour.register(
+        "export_tour_admin",
+        {
+            test: true,
+            url:
+                "/web#model=ir.model.module&view_type=list&cids=&action=base.open_module_tree",
```
Comment (victoralmau) [inline web_disable_export_group/static/src/js/web_disable_export_group_tour.js]:
```suggestion
                "/web#model=ir.ui.view&view_type=list&cids=&action=base.action_ui_view",
```

File: web_disable_export_group/static/src/js/web_disable_export_group_tour.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                    '")',
+            },
+        ]
+    );
+    tour.register(
+        "export_tour_demo",
+        {
+            test: true,
+            url:
+                "/web#model=ir.model.module&view_type=list&cids=&action=base.open_module_tree",
```
Comment (victoralmau) [inline web_disable_export_group/static/src/js/web_disable_export_group_tour.js]:
```suggestion
                "/web#model=ir.ui.view&view_type=list&cids=&action=base.action_ui_view",
```

File: web_disable_export_group/static/src/js/web_disable_export_group_tour.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        },
+        [
+            {
+                content: "Select all records",
+                trigger: ".custom-control-input:first",
+            },
+            {
+                content: "Open actions",
+                trigger: ".o_dropdown_toggler_btn:eq(1)",
```
Comment (victoralmau) [inline web_disable_export_group/static/src/js/web_disable_export_group_tour.js]:
```suggestion
                trigger: ".o_dropdown_toggler_btn",
```

File: web_disable_export_group/static/src/js/web_disable_export_group_tour.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
```
Comment (victoralmau) [inline web_disable_export_group/static/src/js/web_disable_export_group_tour.js]:
```suggestion
                trigger: ".o_dropdown_toggler_btn",
```

File: web_disable_export_group/static/src/js/web_disable_export_group_tour.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+            },
+            {
+                content: "Check if Export button does not exist",
+                trigger:
+                    '.o_control_panel div.o_dropdown_menu:eq(1) a:first:not(:contains("' +
```
Comment (victoralmau) [inline web_disable_export_group/static/src/js/web_disable_export_group_tour.js]:
```suggestion
                    '.o_control_panel div.o_dropdown_menu a:first:not(:contains("' +
```

File: web_disable_export_group/tests/test_tour.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+                "groups_id": [(4, self.browse_ref("base.group_system").id)],
+            }
+        )
+        self.start_tour("/web", "export_tour_demo", login="demo")
+        # Remove demo user to Administration / Settings group
+        demo_user.update(
+            {
+                "groups_id": [(3, self.browse_ref("base.group_system").id)],
+            }
+        )
```
Comment (victoralmau) [inline web_disable_export_group/tests/test_tour.py]:
```suggestion
        self.start_tour("/web", "export_tour_demo", login="demo")
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Although work fine, maybe change to other model allow to use it in all tour tests and simplify some things (for example, add `group_ids` to demo user and delete it after).

**Comment by joao-p-marques** [comment]
Thank you for the suggestions!

Changes made.

**Review by ferranmateu — APPROVED** [review]
Functional review ok

**Pull request merged**